bin/aapp_dr_runner.py

```diff
@@ -167,8 +162,8 @@ def cleanup_aapp_logfiles_archive(config):
     try:
         directory_list = glob(
             '%s/*' % config['aapp_processes'][config.process_name]['aapp_log_files_archive_dir'])
-        dummy = [delete_old_dirs(s, config['aapp_processes'][config.process_name][
-                                 'aapp_log_files_archive_length']) for s in directory_list if os.path.isdir(s)]
+        _ = [delete_old_dirs(s, config['aapp_processes'][config.process_name][
+            'aapp_log_files_archive_length']) for s in directory_list if os.path.isdir(s)]
```

Comment:
Is this using a list comprehension purely for the side effects?  I believe it's discouraged to use list comprehensions with side effects at all, even when using the result.  If you're not using the result, it may be clearer when written as a regular for loop block.

Reply:
I commented on that too :)

Reply:
Thanks, I didn't notice this code at all. But it is hard to read as well. Thx @mraspaud for the proposal you gave, I took it!